sound/soc/intel/boards/sof_realtek_common.c

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: GPL-2.0-only
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+#include <sound/pcm_params.h>
+#include <sound/soc.h>
+#include "../../codecs/rt1011.h"
+#include "sof_realtek_common.h"
+
+
+static const struct snd_soc_dapm_route rt1011_dapm_routes[] = {
+	/* speaker */
+	{ "Left Spk", NULL, "Left SPO" },
+	{ "Right Spk", NULL, "Right SPO" },
```

Comment:
You don't need to separate "Left Spk" and "Right Spk" unless you want to control them separately. See https://github.com/thesofproject/linux/pull/2585

Reply:
maybe we can keep this flexibility to ucm?

Reply:
> maybe we can keep this flexibility to ucm?

Ok, let's keep the flexibility.